content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -79,42 +79,62 @@ With Istio, you can enable authentication for end users through [request authent
     <redacted>
     {{< /text >}}
 
-## Authorization is too restrictive
-
-When you first enable authorization for a service, all requests are denied by default. After you add one or more authorization policies, then
-matching requests should flow through. If all requests continue to be denied, you can try the following:
+## Authorization is too restrictive or permissive
+
+### Make sure there are no typo in the policy YAML file
```

Comment:
It does seem that removing the numbering in this one section makes it out of place. If you compare the existing istio.io page with the page from your `deploy` test, you can see the differences.

ex:  https://istio.io/latest/docs/ops/common-problems/security-issues/
compared to: https://deploy-preview-9043--preliminary-istio.netlify.app/latest/docs/ops/common-problems/security-issues/

I do like that the sections are now called out in the index (and indented).

Reply:
not sure if you want any other changes for this?